index.js

```diff
@@ -29,6 +29,7 @@ function fastProxy (opts = {}) {
 
   const cache = getCacheStorage(opts.cacheURLs)
   const base = opts.base
+  const queryStringModule = opts.queryString || querystring
```

Comment:
```suggestion
  const queryStringModule = opts.queryString || require('fast-querystring')
```

require only when needed.

Reply:
Nice optimization. Thanks!